src/printer.js

```diff
@@ -181,6 +181,18 @@ function genericPrintNoParens(path, options, print, args) {
     case "EmptyStatement":
       return "";
     case "ExpressionStatement":
+      // Detect Flow-parsed directives
+      if (n.directive) {
+        return concat([
+          nodeStr(
+            n,
+            Object.assign({}, options, {
```

Comment:
`options` here are the global ones we set by prettier. I was just asking for you to add a third argument when it's a flow directive literal.